app/views/layouts/application.html.erb

```diff
@@ -136,7 +136,7 @@
               </li>
               <% else %>
               <li class="usa-nav__primary-item">
-                <%= link_to t('links.sign_in'), '/auth/logindotgov', method: :post, class: 'usa-nav__link usa-current' %>
+                <%= button_to t('links.sign_in'), '/auth/logindotgov', method: :post, class: 'usa-button usa-current' %>
```

Comment:
I am 90% certain we have USWDS styles that can format as a plain link? but i don't feel strongly

Reply:
If we do, that'd be great!  I need something similar for some IDP work.

Reply:
Confirmed: [`.usa-button--unstyled`](https://designsystem.digital.gov/components/button/) , [we use it a bunch in the IDP](https://github.com/18F/identity-idp/search?q=usa-button--unstyled)